conda_smithy/templates/circle.yml.tmpl

```diff
@@ -17,8 +19,16 @@ dependencies:
   override:
     - docker pull condaforge/linux-anvil
 
+{% endif -%}
+{% endblock -%}
 test:
   override:
+{%- block env_test %}
+{%- if matrix %}
     # Run, test and (if we have a BINSTAR_TOKEN) upload the distributions.
     - ./ci_support/run_docker_build.sh
-{# Keep the terminal newline. #}
+{% else %}
+    # In case the skip fails for some reason, do a fast finish.
```

Comment:
Sorry, I only meant to indicate that the comment wasn't helpful in my case and my guess is that a person using conda smithy as a casual user will have th same problem. Maybe:

"The Circle-CI build should not be active but if this is not true for some reason, do a fast finish."

?